runtime/service_integ_test.go

```diff
@@ -277,9 +277,10 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 
 			rootfsPath := defaultVMRootfsPath
 
+			vmIDStr := strconv.Itoa(vmID)
 			fcClient := fccontrol.NewFirecrackerClient(pluginClient.Client())
 			req := &proto.CreateVMRequest{
-				VMID: strconv.Itoa(vmID),
+				VMID: vmIDStr,
```

Comment:
Since CreateVM now returns a response with data is it worth adding tests around that in this file?

Reply:
Ah good call, I changed the `TestMultipleVMs_Isolated` to check the cgroup path in the response instead of one the jailer.